stable/external-dns/values.yaml

```diff
@@ -0,0 +1,79 @@
+
+## Details about the image to be pulled.
+image:
+  repository: registry.opensource.zalan.do
+  name: teapot/external-dns
+  tag: v0.3.0
+  pullPolicy: IfNotPresent
+
+## This controls which types of resource external-dns should 'watch' for new
+## DNS entries.
+source:
+  - service
+  - ingress
+
+## The Kubernetes API server to connect to, leave unset to use auto-detect.
+# master: http://127.0.0.1:8080
+
+## Limit sources of endpoints to a specific namespace.
+# namespace: all
+
+## A templated string that's used to generate DNS names
+# fqdnTemplate: {{.Name}}.my-org.com
+
+## Process annotation semantics from legacy implementations (options: mate, molecule )
+# compatibility: 
+
+## The DNS provider where the DNS records will be created (options: aws, google, inmemory, azure )
+provider: aws
+
+## When using the Google provider, specify the Google project (required when --provider=google)
+# googleProject:
+
+## Limit possible target zones by a domain suffix (optional)
+# domainFilter:
+
+## When using the Azure provider, override the Azure resource group to use (optional)
+# azureResourceGroup:
+
+## Modify how DNS records are sychronized between sources and providers (options: sync, upsert-only )
+# policy: sink
+
+## The registry implementation to use to keep track of DNS record ownership (default: txt, options: txt, noop )
+registry: txt
+
+## When using the TXT registry, a name that identifies this instance of ExternalDNS (default: default)
+txtOwnerId: my-identifier
```

Comment:
Since we're using helm could we have a smarter default value here? maybe the release name?